packages/components/src/popover/index.tsx

```diff
@@ -492,7 +543,9 @@ function PopoverSlot( { name = SLOT_NAME }, ref ) {
 	);
 }
 
-PopoverContainer.Slot = forwardRef( PopoverSlot );
-PopoverContainer.__unstableSlotNameProvider = slotNameContext.Provider;
+// @ts-expect-error For Legacy Reasons
+Popover.Slot = forwardRef( PopoverSlot );
+// @ts-expect-error For Legacy Reasons
+Popover.__unstableSlotNameProvider = slotNameContext.Provider;
 
-export default PopoverContainer;
+export default Popover;
```

Comment:
An alternative to this could be to:

- move the `Popover` component from `index.tsx` to `component.tsx`
- use `index.tsx` to export the `Slot`, `__unstableSlotNameProvider` and `default` exports — something like:

```tsx
export function Slot (
	{ name = SLOT_NAME }: { name?: string },
	ref: ForwardedRef< any >
) {
	return (
		<Slot
			// @ts-expect-error Need to type `SlotFill`
			bubblesVirtually
			name={ name }
			className="popover-slot"
	);
}

export const __unstableSlotNameProvider = slotNameContext.Provider;

export {Popover as default} from './component.tsx`;
```


Thoughts? Could it work?